src/FormatNumber.elm

```diff
@@ -113,13 +120,23 @@ format locale num =
                     -truncated
                         |> toFloat
                         |> format { locale | decimals = 0 }
-                        |> String.cons '−'
+
+        decimals : String
+        decimals =
+            if locale.decimals == 0 then
```

Comment:
Now we have this `if locale.decimals == 0 then` inside the `let` block and then again in the `in` block. Is it possible to avoid this repetition and implement this logic just in one place (probably the `in` block)?